services/common/src/constants/strings.tsx

```diff
@@ -108,6 +109,11 @@ export const MINE_REPORTS_TYPE = {
   tailingsReports: "TAR",
 };
 
+export enum MINE_REPORTS_ENUM {
```

Comment:
We do have the `services/common/src/constants/enums.ts` which might be a better place for this to live.

Reply:
Yeaah I was debating the location because the MINE_REPORTS_TYPE is defined right above it, but enums is probably better. I think it's going to be a bit gross unless I overhaul the whole thing (not today...) I think I'll make an enum for MINE_REPORTS_TYPE too.

Reply:
Yeah, it actually stuck out because I'm working on an reverse of this enum:

```export enum MineReportType {
  "code-required-reports" = "CRR",
  "permit-required-reports" = "PRR",
  "tailings-reports" = "TAR",
}```

Used for routing purposes.

Reply:
I am literally about to commit it, but I went for the "use as few letters as possible" approach and came up with:
```
CRR = "CRR",
PRR = "PRR",
TAR = "TAR",
```

Debatable whether that's a good idea (that is, completely non-descript labels, that will make no sense if the value ever does change).... I suppose I'm putting it right under MINE_REPORT_SUBMISSION_CODES that does the same thing, and above MINE_INCIDENT_DOCUMENT_TYPE_CODE which.... also does the same thing. Hmm. I can see why you want that format for routing purposes.

It almost wants a whole object or something
```
REPORT_TYPES = {
  crr: {code: "CRR", "label: "Code Required Reports", route: "code-required-reports"},
  ...etc
}
```
But it felt like overkill. And then surely there'll be a mockup with "Reports required by the code" or some other non-matchy scenario just to make it useless and/or extremely convoluted. Maybe something to discuss at some point.

Reply:
Hmmm, I'm kind of with you there.  It feels clunky to have 2-3 different strings/enums to handle this one thing.  I think the idea of the object might be the cleanest solution.